ui/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @fileoverview Utility functions related to the UI.
+ */
+
+
+/**
+ * Creates a style element and inserts it into the head of the document.
+ * @param {Array.Object} rules A list of rules to insert into the created
+ *     style. Rules must contain a `selector` key, and a `declarations` key,
+ *     which is a list. Example:
+ *     
+ *     [{
+ *         selector: 'body',
+ *         declarations: [
+ *             'font-weight: bold',
+ *             'color: red'
+ *         ]
+ *      }].
```

Comment:
Just a thought, how would you feel about having it look like regular CSS, using a map for everything? Something like:

```javascript
ui.createStyle({
  'body': {
    'font-weight': 'bold',
    'color': 'red'
  }
});
```